tinygrad/tensor.py

```diff
@@ -1962,7 +1968,7 @@ def square(self):
     return self*self
   def clip(self, min_, max_):
     """
-    Clips (limits) the values in the tensor between `min_` and `max_` element-wise.
+    Clips (clamps) the values in the tensor between `min_` and `max_` element-wise.
```

Comment:
is limits not better here? tho i guess both work

Reply:
in torch, `clip` is an alias of `clamp`